ports/esp32/boards/deploy.md

```diff
@@ -1,14 +1,44 @@
-Program your board using the esptool.py program, found [here](https://github.com/espressif/esptool).
+Program your board using the esptool.py program, found [here](https://docs.espressif.com/projects/esptool/en/latest/{mcu}/).
+
+### Erasing
 
 If you are putting MicroPython on your board for the first time then you should
 first erase the entire flash using:
 
 ```bash
-esptool.py --chip esp32 --port /dev/ttyUSB0 erase_flash
+esptool.py --chip {mcu} erase_flash
```

Comment:
I wasn't sure what to do with the `--chip` argument here but I settled on keeping it for the erase_flash command (one less chance of erasing wrong chip), but omitting it for write_flash command (esptool.py will error out if it detects the .bin file doesn't match the target chip, anyway).